src/module.ts

```diff
@@ -25,13 +28,26 @@ export default defineNuxtModule<ModuleOptions>({
     siteKey: nuxt.options.dev ? '1x00000000000000000000AA' : undefined,
     addValidateEndpoint: false,
   }),
-  async setup (options, nuxt) {
+  async setup(options, nuxt) {
     const siteKey = options.siteKey || nuxt.options.runtimeConfig.public?.turnstile?.siteKey
     if (!siteKey) {
       console.warn('`@nuxtjs/turnstile` is disabled as no site key was provided.')
       return
     }
 
+    const secretKeyPath =
+      options.secretKeyPath || nuxt.options.runtimeConfig.turnstile?.secretKeyPath
```

Comment:
we should not support setting `secretKeyPath` in the runtime config as this may give a misleading impression that you can set this key at runtime, which will not work - this module only runs at build time

Reply:
Good point! I moved the path from the runtimeConfig to the inlineOptions. Hope this is good now :)

Reply:
As I see the suggested change implemented I'll break the etiquette of letting the thread owner resolve their thread once and mark this conversation as resolved. I hope that saves you some time, Daniel, but feel free to reopen! 🤙